control_panel_api/tests/test_models.py

```diff
@@ -15,8 +15,33 @@
 from control_panel_api.tests import APP_IAM_ROLE_ASSUME_POLICY
 
 
-class MembershipsTestCase(TestCase):
+class UserTestCase(TestCase):
+    @patch('control_panel_api.services.create_user_role')
+    def test_aws_create_role_calls_service(self, mock_create_user_role):
+        username = 'james'
+        user = User.objects.create(username=username)
+        user.aws_create_role()
+
+        expected_role_name = f"test_user_{username}"
+
+        mock_create_user_role.assert_called_with(
```

Comment:
Minor annoying note, I may be wrong here but we can probably now squeeze these `assert_called_with` params in a single line without breaking the our holy 80-chars rule 🚓 :D

Reply:
More than happy to oblige!